test/fail_compilation/fail19890a.d

```diff
@@ -1,8 +1,7 @@
 /*
-PERMUTE_ARGS:
 TEST_OUTPUT:
 ---
-fail_compilation/fail19890a.d(9): Error: `void[$n$$?:64=LU$]` size 1 * $n$ exceeds $?:windows+32=0x1000000|0x7fffffff$ size limit for static array
+fail_compilation/fail19890a.d(8): Error: `void[18446744073709551615LU]` size 1 * 18446744073709551615 exceeds 0x7fffffff size limit for static array
```

Comment:
Nope

Reply:
I've just gotten round to looking at the test results.  AUTO_UPDATE=1 does a rubbish job at replacing regexes.

Reply:
CC @MoonlightSentinel what do you think of simply ignoring test outputs with an regex for the auto updating?

Reply:
Thought the same, it's impossible to automate those.